templates/index.html

```diff
@@ -7,6 +7,7 @@
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="mobile-web-app-capable" content="yes">
   <script src="https://epschedule-static.storage.googleapis.com/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script> <!-- gets icons -->
```

Comment:
If we need a special-case icon we can bring it in similar to how the calendar icon is done below with iron-iconset-svg. (Or we can try using the "last-page" icon as noted below.)

Reply:
last-page doesn't work, and i tried to get anything working and none of it works. I pushed all the other edits. can we just use this icon for now?

Reply:
that's fine. I'll see if I can figure out how to get this to work later